### PR TITLE
adding info on database population

### PR DESCRIPTION
**Pull request by pmcote**
Let me know if something here doesn't make sense, my brain is pretty non-functional tonight :sleeping:

### CODE REVIEW

File: lessons/05-flex-burger-help/INCLASS.md [hunk 1]
```diff
@@ -7,9 +7,30 @@
...
+####Faking It 
+If you just want data in your database so you can play around with formatting your Handlebars templates or JQuery selectors, it is really easy to fake what your data would look like by creating an object to pass to Handlebars. The one caveat with this approach is that your database output might not look exactly like you expect, but once you have something that works with Handlebars, you can format the data from the database in the same pattern.
+
+To implement this, you just pass your temporary object to Handlebars in res.render:
+
+```javascript
+res.render(‘ingredients’, {ingredients : [
+  {name: 'Tomato',
+   price: '1' },
+  {name: 'Siracha',
```
Comment (benkahle) [inline lessons/05-flex-burger-help/INCLASS.md:24]: Sriracha :)

File: lessons/05-flex-burger-help/INCLASS.md [hunk 2]
```diff
@@ -7,9 +7,30 @@
...
+   price: '2' }
+]});
+```
+
+Now you can play around with your fake data on the front end of your app until you are ready to move forward to working with a database. 
+
+####Create An Add Route
+You will eventually need a way to add initial information to your database for real. You can think of this method as what you did in your cat app with the ‘new cat’ route. For the purpose of populating your database, you can make an array of random ingredients and a random number generator and have your app save that as ingredient data every time you go to that route. Alternatively, you can start by implementing the addIngredient’s route on the front end and have the data you enter into a form POST to the server and populate your database. Hint: this should look a lot like the POST route from the class five readme, but with the added the functionality of saving the data you receive in req.body to your database.
```
Comment (benkahle) [inline lessons/05-flex-burger-help/INCLASS.md:32]: last sentence:
"Hint: This"
"but with the added functionality"

File: lessons/05-flex-burger-help/INCLASS.md (hunk 1 above)
Comment (sarahwalters) [inline lessons/05-flex-burger-help/INCLASS.md:24]: fixed when I merged

File: lessons/05-flex-burger-help/INCLASS.md (hunk 2 above)
Comment (sarahwalters) [inline lessons/05-flex-burger-help/INCLASS.md:32]: fixing in a separate PR now

**Pull request merged**